### PR TITLE
Fix event modify unavailable day bug

### PR DESCRIPTION
**Pull request by tuomas777**
Fixed a bug in event modifying and made one general enhancement for event tests.

### CODE REVIEW

File: events/tests/test_event_api.py [hunk 1]
```diff
@@ -283,3 +298,31 @@ def test_event_cannot_be_created_in_approved_state(api_client, contract_zone):
...
+
+def test_event_cannot_be_created_when_days_are_full(
+    official_api_client, contract_zone, settings
+):
+    events = EventFactory.create_batch(
+        3, start_time=EVENT_DATA["start_time"], end_time=EVENT_DATA["end_time"]
+    )
+    assert all(event.contract_zone == contract_zone for event in events)
+
+    ret = post(official_api_client, LIST_URL, EVENT_DATA, 400)
```
Comment (igordavydsson) [inline events/tests/test_event_api.py]: what does `ret` stand for?

Comment (tuomas777) [reply events/tests/test_event_api.py]: not sure, but it was used earlier in the same file as well :D

Comment (igordavydsson) [reply events/tests/test_event_api.py]: should it be `res` at least? : D

Comment (tuomas777) [reply events/tests/test_event_api.py]: I think it was intentionally not `res` / `response`. ANYWAY, changed it to `response_data` now :D

Comment (igordavydsson) [reply events/tests/test_event_api.py]: I am not against `ret`, just was curious what was meant by it. ANYWAY, looks good now : D

Comment (tuomas777) [reply events/tests/test_event_api.py]: I am, it was stupid.

**Pull request merged**